CHANGES.md

```diff
@@ -1,4 +1,9 @@
 
+## 2.0.0.post1 (2021-02-04)
```

Comment:
Why not just `2.0.1`? I think this counts as a bug fix...

Reply:
yeah I was a bit hesitant to make a `proper` release for just variable renames 🤷‍♂️

Reply:
I think sticking to just `major.minor.bugfix` is simplest. Bug fix releases are "cheap" to make in my opinion